app/templates/preprints/provider/preprint-detail.hbs

```diff
@@ -20,7 +20,7 @@
     {{#if fetchData.isRunning}}
         <div class="preprint-status-component preprint-status-component--skeleton"></div>
     {{else}}
-        {{preprint-status-banner submission=preprint saving=savingAction submitDecision=(action 'submitDecision')}}
+        {{preprint-status-banner submission=preprint saving=savingAction userHasEnteredReview=userHasEnteredReview submitDecision=(action 'submitDecision')}}
```

Comment:
It looks like `preprint-status-banner` doesn't actually use `userHasEnteredReview`, it just sets it to notify the controller of a change. Passing an action here would be a better/cleaner fit for that than a two-way binding. The component could use that action instead of `closeReviewerFeedback`.